### DISCUSSION TITLE
Does this app have collection (similar to postman) support?

### DISCUSSION THREAD

**Discussion by vkhobor** [1buran/rHttp#2] [Q&A] [answered]
.

**Comment by 1buran** [comment, marked as answer]
I doesn't, currently it can only save the current session (request and response with their headers, bodies etc). 

It would be cool to implement something like collections, chains of requests with simple logic of variable substitution, but it requires a lot of effort :) 

Maybe I'll have more free time to add something from this list of wishes or maybe someone else may do this, PR is welcome!
